SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -34,8 +55,7 @@ void SpatialEventTracer::TraceCallback(void* UserData, const Trace_Item* Item)
 
 		if (FPlatformAtomics::AtomicRead_Relaxed(&EventTracer->FlushOnWriteAtomic))
 		{
-			int64_t Flushresult = Io_Stream_Flush(Stream);
-			if (Flushresult == -1)
+			if (Io_Stream_Flush(Stream) == -1)
```

Comment:
Guess -1 is not a #define in worker? :(

Reply:
Nah :( Just references it in the docs, I guess we could make that clearer here though

Reply:
WORKER_RESULT_FAILURE ?